topics:
  - topic: /test/hoge
    hardware_id: hoge-hw
    keycode: HOGE001
    publish_error: true
    invert: false
    timer_update: false
  - topic: /test/fuga
    hardware_id: fuga-hw
    publish_error: false
    invert: true
    timer_update: true
